Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+}
+% page header for pages other than cover page
+\markboth{Project Design Document Standard}%
+{Linden \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
+
+% Initial setup is over, start building the document itself
+\begin{document}
+\maketitle%
+% correct bad hyphenation here, separated by spaces
+\hyphenation{explor-ation}
+
+\begin{abstract}
+  This project will extend off the work of the SPEX propulsion team's cold gas thruster, from the 2017 spring semester.
+  The goal of the project is to create a cold gas thruster that functions well and provides experience and data from performing a cold burn.
+  A well functioning cold gas thruster has no hindrances in fluid flow and produces maximum thrust by means of optimal expansion.
+  The experience learned here will help in the ultimate goal of integrating such a system in a CubeSat for the invitation of atmospheric reentry.
+
+The cold gas thruster will be a closed system which contains all the required regulation and control to sufficiently model a realistic propulsion system for a satellite,
+this does not refer to the dimensions of the system or the material requirements. It will model a realistic propulsion system by maximized thrust and remote initiation of the cold burn.
+
+      % The abstract is a brief summary of the design document. Typically it includes the purpose of the design document, key goals or objectives, and justifications.
+      % Be sure not to confuse the abstract with the introduction.
+      % It is easiest to write the abstract after the rest of the paper has been written.
+      % That way you can choose key information from the sections that you've already completed and string them together in the abstract.
+      % Consider the abstract to be your elevator pitch to anyone reading this design document.
+      % What are they reading?
+      % What is the goal?
+      % Why is it worth my time?
+      % The abstract is what will show up in Google results and other search engines, and what people will read when they are deciding what is worth their time and brain power.
+\end{abstract}
+
+\label{sec:nomenclature}
+\newcommand{\nomunit}[1]{%
+\renewcommand{\nomentryend}{\hspace*{\fill}#1}}
+\renewcommand{\nompreamble}{
+    % If you include mathematical expressions or express variables in the design document, list them with their corresponding definitions here as a list.
+    % The two lines below make it look nice when defining units/values to constants.
+
+    % Note that math terms and non-math terms are separated and alphabetized, regardless of the order in which they are defined. (Recall terms \$like this\$ are in the math environment)
+    % Read more about advanced nomenclature formatting here:\\
+    % \url{https://www.sharelatex.com/learn/Nomenclatures}
+  }
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{PDD}{Project Design Document}
```

Comment:
This is not resolved by [`f6e5c2a`](https://github.com/RIT-Space-Exploration/SPEX-Standard-Proposal/pull/16/commits/f6e5c2a174bdfed4fde5bd8cdf2140cee4c652a9). Good practice is to remove this if you do not use it in the body of the text.